pkg/test/resourcefixture/testdata/basic/apigee/v1beta1/apigeeorganization/dependencies.yaml

```diff
@@ -15,9 +15,12 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Project
 metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: project-${uniqueId}
 spec:
-  name: project-${uniqueId}
+  resourceID: ${TEST_DEPENDENT_ORG_PROJECT_ID}
```

Comment:
This test is in `resourcefixture/` so I don't think it ends up in our docs.  Given that, could we add a comment here about why we want/need a persistent project (instead of a random ephemeral project) here?

Reply:
It was explained to me; just an artifact of our some internal test project limitations!